curriculum/challenges/english/20-upcoming-python/learn-tree-traversal-by-building-a-binary-search-tree/655a4614304cd36031cb4e75.md

```diff
@@ -0,0 +1,62 @@
+---
+id: 655a4614304cd36031cb4e75
+title: Step 28
+challengeType: 20
+dashedName: step-28
+---
+
+# --description--
+
+Assign the result of the `_delete` call to `self.root`.
+
+This is done because the deletion operation might result in a new root (for example, if the node to be deleted is the current root).
```

Comment:
```suggestion
The deletion operation might result in a new root (for example, if the node to be deleted is the current root).

To handle this case, assign the result of the `_delete` call to `self.root`.
```
I would suggest inverting the order of the sentences to explain the context first and then how to implement it in code.